src/main/webapp/i18n/en/exampleSubmission.json

```diff
@@ -39,6 +39,7 @@
             "createNewSubmission": "Create new example submission",
             "editExampleSubmission": "Edit example submission",
             "modelElementHint": "Hint: Model elements which assessment you missed are highlighted in blue.",
+            "textHint": "Hint: Text Blocks that you need to assess are highlighted in blue.",
```

Comment:
```suggestion
            "textHint": "Hint: Text blocks that you need to assess are highlighted in blue.",
```

Reply:
Done in ca3c502ce164b2b56d110d4124d67f3603e5c9ab